x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -397,6 +398,7 @@ private FunctionDefinition[][] functions() {
     private static FunctionDefinition[][] snapshotFunctions() {
         return new FunctionDefinition[][] {
             new FunctionDefinition[] {
+                def(Wait.class, Wait::new, "wait"),
```

Comment:
Could you add a comment here saying "this is for debugging only and should never be moved to the list of production function"?